sdk-tests/src/test/java/io/dapr/it/pubsub/http/PubSubIT.java

```diff
@@ -413,4 +467,13 @@ public void setId(String id) {
       this.id = id;
     }
   }
+
+  public static class ConvertToLong {
+    public Long value;
+
+    public void setVal(Long value) {
+      this.value = value;
+    }
+  }
```

Comment:
Add getter for value, and equals and Dashcode method (Can autogenerate in IntelliJ using Objects.hash etc ).